tests/installation/opensuse_welcome.pm

```diff
@@ -20,7 +20,32 @@ use utils;
 sub run {
     assert_screen("opensuse-welcome");
     assert_and_click("opensuse-welcome-show-on-boot");
+    # Make sure tick box is unchecked - poo#56024
+    for my $retry (1 .. 5) {
+        if (check_screen("opensuse-welcome-show-on-boot")) {
+            assert_and_click("opensuse-welcome-show-on-boot");
+        } else {
+            last;
+        }
+        if ($retry == 5) {
```

Comment:
If the click succeeds on the 5th try, the test still dies.

Better:
```
assert_screen("opensuse-welcome")
for my $retry (1..5) {
  assert_and_click("opensuse-welcome-show-on-boot");
  last unless check_screen("opensuse-welcome-show-on-boot");
  die "Unable to untick 'Show on next startup'" if $retry == 5;
}
```

Reply:
It should be ok now.